src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/veteran-information.js

```diff
@@ -7,60 +7,67 @@ import _ from 'lodash/fp';
 export const schema = {
   type: 'object',
   properties: {
-    first: genericSchemas.genericTextInput,
-    middle: genericSchemas.genericTextInput,
-    last: genericSchemas.genericTextInput,
-    suffix: {
-      type: 'string',
-      enum: suffixes,
+    veteranInformation: {
+      type: 'object',
+      properties: {
+        first: genericSchemas.genericTextInput,
+        middle: genericSchemas.genericTextInput,
+        last: genericSchemas.genericTextInput,
+        suffix: {
+          type: 'string',
+          enum: suffixes,
+        },
+        ssn: genericSchemas.genericNumberAndDashInput,
+        vaFileNumber: genericSchemas.genericNumberAndDashInput,
+        serviceNumber: genericSchemas.genericNumberAndDashInput,
+        birthDate: genericSchemas.date,
+      },
     },
-    ssn: genericSchemas.genericNumberAndDashInput,
-    vaFileNumber: genericSchemas.genericNumberAndDashInput,
-    serviceNumber: genericSchemas.genericNumberAndDashInput,
-    birthDate: genericSchemas.date,
   },
 };
 
 export const uiSchema = {
-  first: {
-    'ui:title': 'Your first Name',
-    'ui:required': () => true,
-  },
-  middle: {
-    'ui:title': 'Your middle Name',
-  },
-  last: {
-    'ui:title': 'Your last Name',
-    'ui:required': () => true,
-  },
-  suffix: {
-    'ui:options': {
-      widgetClassNames: 'usa-input-medium',
+  veteranInformation: {
+    first: {
+      'ui:title': 'Your first Name',
+      'ui:required': () => true,
     },
-  },
-  ssn: _.merge(_.unset('ui:title', ssnUI), {
-    'ui:title': 'Your Social Security number',
-    'ui:required': () => true,
-  }),
-  vaFileNumber: {
-    'ui:title': 'Your VA file number',
-    'ui:errorMessages': {
-      pattern: 'Please enter only numbers',
+    middle: {
+      'ui:title': 'Your middle Name',
     },
-    'ui:options': {
-      widgetClassNames: 'usa-input-medium',
+    last: {
+      'ui:title': 'Your last Name',
```

Comment:
Title casing is inconsistent. These should read `Your first name`, `Your middle name`, `Your last name`.

Reply:
Fixed these